renderdoc/driver/d3d11/d3d11_counters.cpp

```diff
@@ -157,11 +266,12 @@ vector<CounterResult> D3D11DebugManager::FetchCounters(const vector<uint32_t> &c
     return ret;
   }
 
-  uint32_t counterID = counters[0];
-  RDCASSERT(counters.size() == 1);
-  RDCASSERT(counterID == eCounter_EventGPUDuration);
+  //uint32_t counterID = counters[0];
+  //RDCASSERT(counters.size() == 1);
+  //RDCASSERT(counterID == eCounter_EventGPUDuration);
 
-  SCOPED_TIMER("Fetch Counters for %u", counterID);
+  //SCOPED_TIMER("Fetch Counters for %u", counterID);
```

Comment:
You can just delete commented code like this. It won't be relevant again now that we have multiple counters and it's always there in the history if it's needed.